sections/main-cart-items.liquid

```diff
@@ -274,8 +275,9 @@
                             </quantity-input>
                           </div>
                           <cart-remove-button
-                            id="Remove-{{ item.index | plus: 1 }}"
-                            data-index="{{ item.index | plus: 1 }}"
+                            id="Remove-{{ item.key | handle }}"
+                            data-handleized-key="{{ item.key | handle }}"
```

Comment:
The reason it is handleized here is because the key contains the ":" character, which is invalid character when used as a CSS selector.